### DISCUSSION TITLE
Add a new "Delay slot notch" checkbox to the debugger window interface of PCSX2

### DISCUSSION THREAD

**Discussion by ghost** [PCSX2/pcsx2#7511] [General]
While this new checkbox is checked:

Add a small visible notch to indicate instructions that are on the delay slot of a branch.

This can be very both useful and helpful to distinguish between instructions that are on the delay slot of a branch and instructions that are not!

Don't you think so?

**Comment by RedDevilus** [comment]
Could you please keep the requests in issues and not make 3 of them in discussions, especially when WX debugger is on track to be purged and be replaced in Qt?

**Reply by ghost** [reply]
> Could you please keep the requests in issues and not make 3 of them in discussions, especially when WX debugger is on track to be purged and be replaced in Qt?

Are you planning to add the debugger to Qt?

**Reply by Mrlinkwii** [reply]
yes https://github.com/PCSX2/pcsx2/pull/6161

**Reply by RedDevilus** [reply]
I'm not the one making it but it's currently in the works:
https://github.com/PCSX2/pcsx2/pull/6161

Please be patient and test the artifacts (checks) and not bombard the creators with messages as to not make them annoyed it is still a volunteering place.

**Reply by ghost** [reply]
> I'm not the one making it but it's currently in the works: #6161
> 
> Please be patient and test the artifacts (checks) and not bombard the creators with messages as to not make them annoyed it is still a volunteering place.

I know that software development in general, especially emulator development, is very hard and I am appreciating your hard work! I am grateful! All what I want to do is to contribute and improve your best PS2 emulator in the world!

When I have a new idea about how PCSX2 can be improved I just want to speak it before I forget!

I have spoken all my ideas and right now I have no new idea to speak so I am going to play something.

Good luck developers!

And I am a patient person. I can wait.
